src/test/blockfilter_tests.cpp

```diff
@@ -112,7 +127,7 @@ BOOST_AUTO_TEST_CASE(blockfilter_basic_test)
 
     CDataStream stream(SER_NETWORK, PROTOCOL_VERSION);
     stream << block_filter;
-    stream >> block_filter2;
+    block_filter2 = UnserializeBlockFilter(stream);
```

Comment:
I'd just initialize at once:
```suggestion
    BlockFilter block_filter2 {UnserializeBlockFilter(stream)};
```